common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -96,6 +96,7 @@
         try:
             return Template("""
                 {% load render_bundle from webpack_loader %}
+                {% render_bundle "commons" %}
```

Comment:
I don't think we have any such sections yet, so it is mostly a concern for the future. You could try cloning a webpack section to render twice, e.g.

https://github.com/edx/edx-platform/blob/master/openedx/features/course_experience/templates/course_experience/course-outline-fragment.html#L163